profile/README.md

```diff
@@ -10,3 +10,45 @@
 🍿 Fun facts - what does your team eat for breakfast?
 🧙 Remember, you can do mighty things with the power of [Markdown](https://docs.github.com/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)
 -->
+
+As “Big Data” cancer researchers, we both apply existing techniques and continuously work to improve them and create new methods to create clinically useful diagnostics that personalize therapy.
+
+Much of our work to date has focused on prostate cancer, but we are increasingly applying the same techniques to cancers of the kidney, bladder and other sites, and discovering fascinating commonalities across them.
+
+***Want to learn more?***
+<br>Go to the [Cancer Data Sciences page](https://www.uclahealth.org/departments/urology/iuo/research/faculty-labs/dr-paul-boutros-lab) where you can
+read about our recent [research areas](https://www.uclahealth.org/departments/urology/iuo/research/faculty-labs/dr-paul-boutros-lab/research-areas) and some of our recent [publications](https://www.uclahealth.org/departments/urology/iuo/research/faculty-labs/dr-paul-boutros-lab/publications).
+
+--- 
+
+## :bar_chart: Tools
+From this organization GitHub page, you can find our **R packages, Docker images, and Nextflow pipelines.** 
+
+Whether you are an external researcher, one of our collaborators, or someone hoping to [join our team](#briefcase-join-our-team), we encourage you to use our tools and create issues for any problems you may encounter. Together with your feedback, we can improve our tools and make them more accessible for the next generation of researchers, who will use these techniques to cure cancer and improve human health and well-being in ways we can’t imagine today!
+
+
+### :star: Most Popular 
+| R Package | Description | Resources | CRAN? |
+| :--------: | ----------- | :--------: | :----: |
+| [BoutrosLab.plotting.general](https://github.com/uclahs-cds/public-R-BoutrosLab-plotting-general) | Contains several plotting functions such as barplots, scatterplots, and heatmaps. Also includes functions to combine plots and assist in the creation of these plots. | [Docs](https://uclahs-cds.github.io/public-R-BoutrosLab-plotting-general/) | [CRAN](https://cloud.r-project.org/web/packages/BoutrosLab.plotting.general/index.html) |
+| [public-R-VennDiagram](https://github.com/uclahs-cds/public-R-VennDiagram)|A set of functions to generate high-resolution Venn and Euler plots. Includes handling for several special cases, including two-case scaling, and extensive customization of plot shape and structure.| [Docs](https://github.com/uclahs-cds/public-R-VennDiagram/tree/main/man) | No |
```

Comment:
VennDiagram is on CRAN https://cran.r-project.org/web/packages/VennDiagram/index.html